### DISCUSSION TITLE
更新日志 - 22.03.5 - 20231215

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#1186] [Announcements]
# 系统
1. luci 支持 iStoreX 的首页替换

# 应用
* QuickStart：更新以支持新版 iStoreX

# 设备
* x86: 集成 sit 驱动
* rk3568:
  1. 支持 GPU
  2. 修复 R5S/R5C 的 HDMI

**Comment by wanggy429** [comment]
请问，下载固件并刷写重启后，仍然提示需要存在之前一模一样的固件更新，是什么问题。

**Reply by jjm2473** [reply]
应该是升级过程失败了，重试一下看看。下个版本固件会加上系统升级的日志存储，用来检查这类升级失败的问题。

**Reply by jjm2473** [reply]
估计是有程序杀不掉，升级程序没法继续。

**Reply by wanggy429** [reply]
试了很多次了，都不行。重启之后再检查升级也是一样的。

**Reply by jjm2473** [reply]
参考这个，看看是不是执行了破坏系统的操作 https://github.com/istoreos/istoreos/issues/1200
